### ISSUE TITLE
Allow for full chain exports and import with stateroots

### ISSUE THREAD

**Issue by travisperson** [filecoin-project/lotus#2342]
In trusted environments it would be hugely beneficial to have the ability to take full chain exports (and import them) with their stateroots to increase speed of new node deployments.

There is a current work around of stopping the daemon, extracting the `$LOTUS_PATH/datastore` and then starting the daemon back up. This works okay, but is going to be error prone to change to the underlying datastore implementation, version, or just the way lotus chooses to store information.

Something like

```
$ lotus chain export --tipset <tipset> chain.car
$ lotus daemon --import-chain chain.car --import-with-stateroot(s)
```

**Comment by symanovich-a** [comment]
@travisperson you are not alone with this enhancement) #147

**Comment by travisperson** [comment]
That PR actually got merged or at least a rewrite of it in https://github.com/filecoin-project/lotus/pull/1128 (and updated in https://github.com/filecoin-project/lotus/pull/1439), so chain exporting works. We just don't have a way to include the stateroots along with them at the moment.

**Comment by travisperson** [comment]
Implemented in #3463
